### PR TITLE
NLog 5.0 RC1 changes the behavior of NLogProviderOptions-CaptureEventId

### PR DESCRIPTION
**Pull request by snakefoot**
Trying to improve the sentences.

### CODE REVIEW

File: _posts/2021-08-25-nlog-5-0-preview1-ready.md [hunk 1]
```diff
@@ -597,11 +597,11 @@ NLog LoggingProvider no longer follows the Microsoft Logger filtering configurat
...
 
-### NLog.Extensions.Logging skips capture of EventId struct
-NLog LoggingProvider has changed from capturing the EventId-struct + EventId_Id-number + EventId_Name to just
-capture EventId-number and EventName-identifier.
+### NLog.Extensions.Logging changes capture of EventId
+NLog LoggingProvider will no longer capture `EventId`-struct + `EventId_Id`-number + `EventId_Name`, instead it will 
+by default capture the properties `EventId` (integer) and `EventName` (string).
 
-* **Impact:** EventId_Id and EventId_Name will not be included by default. Instead it will be EventId and EventName.
+* **Impact:** `EventId_Id` and `EventId_Name` will not be included by default. Instead it will be properties `EventId` and `EventName`.
```
Comment (304NotModified) [inline _posts/2021-08-25-nlog-5-0-preview1-ready.md:604]: Maybe 

`EventId_Id` and `EventId_Name` are not included by default. Those are replaced with the properties `EventId` and `EventName`.

**Review by 304NotModified — APPROVED** [review]
This is indeed better to read. Added a suggestion

File: _posts/2021-08-25-nlog-5-0-preview1-ready.md (hunk 1 above)
Comment (snakefoot) [reply _posts/2021-08-25-nlog-5-0-preview1-ready.md]: Have optimized the sentence a little more. Thanks for the input

**Comment by snakefoot** [comment]
@304NotModified Ready for merge

**Pull request merged**